cilium-dbg/cmd/bpf_lb_reverse_sk.go

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
*(On the commit description itself.)*

Could you wrap the lines at 75 characters?